bentoml/adapters/base_output.py

```diff
@@ -50,7 +50,7 @@ class BaseOutputAdapter:
     such as HTTP response, command line stdout or AWS Lambda event object.
     """
 
-    def __init__(self, cors='*'):
+    def __init__(self, cors=None):
```

Comment:
What do you think making this a config under bento_server instead of a parameter for each API's output adapter?

Reply:
Similarly, it can be a deployment option for AWS lambda deployment down the line, but we can hard code it to "*" for lambda for now, which is the current behavior anyway

Reply:
> What do you think making this a config under bento_server instead of a parameter for each API's output adapter?

LGTM. The CORS option could be literally the property of the service.

```python
@bentoml.aws_lambda_option(cors="*")
@bentoml.http_option(cors=None)
@bentoml.api(...)
def preodict
```

or just

```python
@bentoml.api(cors="*")
def preodict
```

> it can be a deployment option for AWS lambda deployment

Not recommend. To achieve this in deployment, take a reverse proxy like Nginx would be better.

Reply:
@parano reply updated.